### ISSUE TITLE
"ERROR: 'NoneType' object has no attribute 'items'" when `--headers` not used (+ fix)

### ISSUE THREAD

**Issue by follower** [nlitsme/zipdump#5]
Thanks for developing & sharing this tool--partial zip file downloading is functionality I've wanted on a number of occasions, so it's great to have discovered this tool. :) 

I encountered an issue when using the tool which appears to be related to the recently added `--headers` option https://github.com/nlitsme/zipdump/commit/887e08151175eabe76694485fa927aeec32cf9dc.

Further details & a fix are in the PR I just opened: https://github.com/nlitsme/zipdump/pull/4

Thanks again for the tool & your time.

**Comment by follower** [comment]
For context, in case it's of interest, on this occasion I'm using `zipdump` to avoid having to download an entire ~750MB zip of the Godot game engine's "export templates" when I only want one ~7.5MB file from it. :)

(I finally went looking for a tool like `zipdump` because Godot is uploading a new beta/release candidate about once a day at the moment. :D )

FWIW a couple of other random notes from my use of `zipdump`:

 * It wasn't immediately clear to me that I also needed to supply `--quick` when *saving* a file in order to avoid the more thorough scan.
 * A release tag would make linking to a specific version of `zipdump` easier when pointing others to it. But I also recognize it's "yet another thing" to do. :)

Anyway, thanks again for sharing the tool--my bandwidth & disk space & I all appreciate it. :)
